src/main/java/humi/DeadlineTask.java

```diff
@@ -0,0 +1,31 @@
+package humi;
+
+public class DeadlineTask extends Task {
+    public String deadline;
+    DeadlineTask(String description, String deadline) {
+        this.taskType = TaskType.DEADLINE;
+        this.description = description;
+        this.isDone = false;
+        this.deadline = deadline;
+
+        System.out.println(Humi.LINE);
+        System.out.println("     Got it. I've added this task:");
```

Comment:
Try not to use magic literals e.g. magic numbers and magic string. Consider giving it a meaning by creating a constant and use the constant in your code.